### DISCUSSION TITLE
Trying out Mimir - skipped partial block and missing data that reappear the day after

### DISCUSSION THREAD

**Discussion by Danne-J** [grafana/mimir#3444] [Help and support]
Hi
have a 3 node cluster in Docker swarm with a consul to tie them up.
cant seem to find a way to use members to get them in a cluster.
Anyway i got some error i cant seem to find answer to, except an old Cortex post refering to config thats not applicable in Mimir

level=warn ts=2022-11-14T10:06:07.492190496Z caller=updater.go:113 component=cleaner user=anonymous msg="skipped partial block when updating bucket index" block=01GHAD541DF0F46ESW8JWPSRRC

Br

**Comment by Danne-J** [comment]
don't know if its related

![image](https://user-images.githubusercontent.com/20092339/201653236-99649fe6-b4b4-4acc-8d3e-8a5e37a71e04.png)

see in the graf the gap of points. that happens everyday but the day after the graf is okej.
Is it the querier, ingester or store-gateway thats need config attention?

**Reply by pracucci** [reply]
Are you running Mimir with the default configuration or have you customise it?
